lib/node-token-validation/src/config/Configuration.ts

```diff
@@ -55,12 +53,9 @@ const DEFAULT_SYSTEM_OPTIONS = {
  * @param {@link (Configuration:type)} 
  * @returns 
  */
-export function buildConfiguration({
-    auth,
-    system,
-}: Configuration): TokenValidationConfiguration {
+export function buildConfiguration(config?: Configuration): TokenValidationConfiguration {
     return {
-        auth: { ...DEFAULT_TOKEN_VALIDATION_OPTIONS, ...auth },
-        system: { ...DEFAULT_SYSTEM_OPTIONS, ...system },
+        auth: { ...DEFAULT_TOKEN_VALIDATION_OPTIONS, ...config?.auth },
```

Comment:
Will this not throw if config is undefined?

Reply:
It doesn't seem to throw. Config is an optional parameter.